Docs.Site/Docs/Reference/Authentication-Modules.markdown

```diff
@@ -0,0 +1,205 @@
+
+Back in the MyGet **Add authentication module** dialog:
+
+* Copy/paste the *Microsoft Account application id* that is displayed under **[My Applications - Converged applications](https://apps.dev.microsoft.com/)**.
+* Click **Add** to save the configuration.
+
+## StackExchange
+
+To allow users to login to MyGet using their StackExchange account, enable the StackExchange authentication module.
+
+Once the **Add authentication module** dialog is shown, we can find **Settings for StackExchange**. Make sure to keep this window open as we will need it for the remainder of the configuration steps.
+
+In a separate browser window:
+
+* Log in to your StackExchange account and open **[Register a new app](https://stackapps.com/apps/oauth/register)**.
+* In the *Application Name* field, enter an application name (e.g. *MyGet*).
+* In the *Description* field, enter an description.
+* In the *OAuth domain* field, paste the domain that is displayed in MyGet *Settings for StackExchange*.
+* In the *Application website* field, paste the application website that is displayed in MyGet *Settings for StackExchange*.
+* Click the **Register Your Application** button. StackExchange will generate the credentials required to configure MyGet.
+
+Back in the MyGet **Add authentication module** dialog:
+
+* Copy/paste the *StackExchange client id*, *StackExchange client secret* and *StackExchange key*.
+* Click **Add** to save the configuration.
+
+## WS-Federation (ADFS)
+
+To allow users to login to MyGet using their ADFS account, enable the WS-Federation authentication module.
+
+Once the **Add authentication module** dialog is shown, we can find **Settings for WS-Federation (ADFS)**. Make sure to keep this window open as we will need it for the remainder of the configuration steps.
+
+On your ADFS server (guide based on [Microsoft Docs - Create a Relying Party Trust](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/operations/create-a-relying-party-trust)):
+
+* In **Server Manager**, click **Tools**, and then select **AD FS Management**.
+* Under **Actions**, click **Add Relying Party Trust**.
+* On the *Welcome* page, choose **Claims aware** and click **Start**.
+* On the *Select Data Source* page, click **Enter data about the relying party manually**, and then click **Next**.
+* On the *Specify Display Name* page, enter a display name (e.g. *MyGet*). Optionally a description can be added as well. Click **Next**.
+* On the *Configure Certificate* page, click **Next**
+* On the *Configure URL* page, enable the* Enable support for the WS-Federation Passive protocol* checkbox. In the *Relying party WS-Federation Passive protocol URL* field, paste the reply URL that is displayed in MyGet *Settings for WS-Federation (ADFS)*. Click **Next**.
+* On the *Configure Identifiers page*, specify two identifiers for this relying party:
+  * The sign-on URL that is displayed in MyGet *Settings for WS-Federation (ADFS)*
```

Comment:
It's `log on` or `sign in`, where does `sign-on` come from? :)

Reply:
From the form they see in front of them on MS website :-)